diff --git a/lib/generators/asset-types/svg.js b/lib/generators/asset-types/svg.js
index 8dcc6bce763d0ed174aa48048d591331f9d61b8c..9848176e7b9c98f14f05ca9fe4551b0bf53d54dc 100644
--- a/lib/generators/asset-types/svg.js
+++ b/lib/generators/asset-types/svg.js
@@ -16,7 +16,7 @@ const generator = {
             .on('end', () => resolve(font.toString()));
         for (const { id, absolutePath } of Object.values(assets)) {
             const glyph = (0, fs_1.createReadStream)(absolutePath);
-            const unicode = String.fromCharCode(codepoints[id]);
+            const unicode = String.fromCodePoint(codepoints[id]);
             glyph.metadata = { name: id, unicode: [unicode] };
             fontStream.write(glyph);
         }
